applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -410,6 +422,8 @@ static void cell_pos_req_wk(struct k_work *work)
 			err = nrf_cloud_location_request(&cell_data, NULL, true, NULL);
 			if (err) {
 				LOG_ERR("Failed to request MCELL, error: %d", err);
+				rsp_send("\r\n#XCELLPOS: %d\r\n", err);
+				run_type = RUN_TYPE_NONE;
```

Comment:
What kind of error codes are replied to the AT-command? They are not listed in the documentation.